prompt/select.ts

```diff
@@ -99,6 +146,167 @@ export class Select<S extends SelectSettings = SelectSettings>
     return this.options[this.listIndex]?.value ?? this.settings.default;
   }
 
+  protected async submit(): Promise<void> {
+    const itemToSubmit = this.options[this.listIndex];
+
+    if (Select.itemIsBackButton(itemToSubmit)) {
+      const previousLevel = this.#parentCategories.pop();
+      if (typeof previousLevel === "object") {
+        this.options = previousLevel.options;
+        if (this.isCurrentlyInsideCategory()) {
+          this.listIndex = 2;
```

Comment:
When you're in a category, the first two items are the back button and a separator, so the default index should skip those elements.

Reply:
I combined the backbutton and the breacrumb now. I think it looks more clean. What do you think?